app/api/models/statistics.py

```diff
@@ -111,12 +111,15 @@ class GenomeStatistics(BaseModel):
     regulation_stats: Regulation
 
     def __init__(self, **data):
-        data["_compiled_data"] = {
-            stats_item["name"]: stats_item["statisticValue"]
-            for stats_item in data["_raw_data"]
-            if stats_item["statisticValue"] != "null"
-        }
-
+        data["_compiled_data"] = {}
+        try:
+            for stats_item in data["_raw_data"]:
+                try:
+                    data["_compiled_data"][stats_item["name"]]=stats_item["statisticValue"]
```

Comment:
Can we use .get() to avoid the try: except and to help the code to survive in these cases?

Reply:
That's reasonable thing to do but keeping try..except will give us an opportunity to find out which fields value is missing. and it will survive the missing fields.

Reply:
I think some of these try excepts are not allowing the code to have standard outputs that can help easily identify the issues specially when we're on k8s. For example: I couldn't even see that GRPC is broken at the moment and everything seem to be fine in the logs except a 500 Error!
I won't get any sot of useful information in my logs unless I remove try: excepts to see where the code is exactly. 
Keep that in mind that in the next update I'm going to change this in web_metadata_api but we can go ahead with this for now.